app/models/account.rb

```diff
@@ -85,7 +81,7 @@ def self.update_if_needed!(account, user)
   # @return [Hash]
   #
   def self.account_attrs_from_user(user)
-    { idme_uuid: user.uuid, edipi: user.edipi, icn: user.icn, sec_id: user.sec_id }
+    Hash[%i[idme_uuid sec_id edipi icn].map { |k| [k, user.send(k)] }]
```

Comment:
I ❤️ this from terseness standpoint, but might be more readable to do:

{
idme_uuid: user.idme_uuid,
sec_id: user.sec_id,
edipi: user.edipi,
icn: user.icn
}